lib/dahlia_data_models/models/listing.rb

```diff
@@ -74,8 +74,10 @@ class Listing < ApplicationRecord
     { custom_api: 'Reserved_community_minimum_age', domain: 'reserved_community_minimum_age', salesforce: 'Reserved_community_minimum_age' },
     { custom_api: 'reservedDescriptor', domain: 'reserved_descriptor', salesforce: 'reservedDescriptor' },
     { custom_api: 'SASE_Required_for_Lottery_Ticket', domain: 'sase_required_for_lottery_ticket', salesforce: 'SASE_Required_for_Lottery_Ticket' },
+    { custom_api: 'showUnitFeatures', domain: 'show_unit_features', salesforce: 'showUnitFeatures' },
```

Comment:
@callaghanc I had to add this and the other new field to these field mappings in order for the Listings site to work. (I didn't realize the Listing site's Angular views are still using the Salesforce API field names.)